addons/mail/static/src/models/messaging_menu.js

```diff
@@ -113,6 +121,26 @@ registerModel({
         _computeNotificationListView() {
             return this.isOpen ? insertAndReplace() : clear();
         },
+        /**
+         * Closes the menu when clicking outside, if appropriate.
+         *
+         * @private
+         * @param {MouseEvent} ev
+         */
+        _onClickCaptureGlobal(ev) {
```

Comment:
I wonder about the private status of this handler. I guess it's okay for global click.